src/lib/libvkdt.h

```diff
@@ -0,0 +1,23 @@
+#ifndef LIBVKDT_H
```

Comment:
the rest of the code base uses `#pragma once` instead.

Reply:
Agree.